cmd/thanos/receive.go

```diff
@@ -36,6 +39,10 @@ func registerReceive(m map[string]setupFunc, app *kingpin.Application, name stri
 	dataDir := cmd.Flag("tsdb.path", "Data directory of TSDB.").
 		Default("./data").String()
 
+	externalLabels := cmd.Flag("external-labels", "External labels to announce. Comma separated list of key=value pairs. This flag will be removed in the future when handling multiple tsdb instances is added.").PlaceHolder("key1=value1,key2=value2").String()
+
+	objStoreConfig := regCommonObjStoreFlags(cmd, "", false)
```

Comment:
Isn't this required? Basing of fact that retention is not configurable?

Reply:
Not a blocker,

Reply:
I could see people just caring about a single instance of this with just local storage. I can see scenarios where it would not have to be set.

Reply:
As a side note, retention should be configurable. People should be able to make a choice how they want local storage to be treated when running receive I think